sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -163,14 +200,6 @@ def get_pm_all(self):
         SOPROC: unit in krad/s
         MER:    unit in dB
         '''
-        # When raised by the host, causes the module to freeze and hold all 
```

Comment:
@jaganbal-a can you raise a PR in platform daemon to fix this ? Because if I merge this PR, then it will break existing platform using ZR.

Reply:
It will not break any functionality. the Freeze/unfreeze done for resetting the statistics register. 
Now the Freeze and unfreeze will be done by the PM manager and PM mgr is default on all platform.